site/pages/cookies.tsx

```diff
@@ -0,0 +1,144 @@
+                                        name="tracking"
+                                        value="on"
+                                        defaultChecked={trackingDefaultValue === "on"}
+                                    />
+                                    <label
+                                        className="govuk-label govuk-radios__label"
+                                        htmlFor="accept-analytics-cookies"
+                                    >
+                                        Use cookies that measure my website use
+                                    </label>
+                                </div>
+                                <div className="govuk-radios__item">
+                                    <input
+                                        className="govuk-radios__input"
+                                        type="radio"
+                                        id="decline-analytics-cookies"
+                                        name="tracking"
+                                        value="off"
+                                        defaultChecked={trackingDefaultValue === "off"}
+                                    />
+                                    <label
+                                        className="govuk-label govuk-radios__label"
+                                        htmlFor="decline-analytics-cookies"
+                                    >
+                                        Do not use cookies that measure my website use
+                                    </label>
+                                </div>
+                            </div>
+                        </fieldset>
+                    </div>
+                    <div>
+                        <h2 className="govuk-heading-m">Strictly necessary cookies</h2>
+                        <p className="govuk-body">These essential cookies are used to remember your progress.</p>
+                        <p className="govuk-body">They always need to be on.</p>
+                        <p className="govuk-link govuk-!-font-size-19">
+                            <a className="govuk-link govuk-!-font-size-19" href="/cookieDetails">
+                                Find out more about our usage of cookies in our cookie policy
+                            </a>
+                        </p>
+                    </div>
+                    <button
+                        className="govuk-button mt-4"
+                        data-module="govuk-button"
+                        id="save-cookie-preferences-button"
+                    >
+                        Save Changes
+                    </button>
+                </>
+            </CsrfForm>
+        </div>
+    </TwoThirdsLayout>
+);
+
+export const getServerSideProps = (ctx: NextPageContext): { props: CookiePreferencesProps } => {
+    const csrfToken = getCsrfToken(ctx);
+    const cookies = parseCookies(ctx);
```

Comment:
Can we use cookies-next in place of nookies here? Then we could remove nookies completely

Reply:
switched to nookies as discussed